backend/organization/models/organization.py

```diff
@@ -0,0 +1,151 @@
+from django.db import models
+
+
+class OrganizationType(models.Model):
+    name = models.CharField(
+        help_text="Points to name of the type. e.g: NGO",
+        verbose_name="Name",
+        max_length=512
+    )
+
+    key = models.CharField(
+        help_text="Points to key of the type.",
+        verbose_name="Key",
+        max_length=512,
+        null=True,
+        blank=True
+    )
+
+    # TODO (Dip): Confirm about additional info column.
```

Comment:
As suggested on slack, I would add an `additionalInfo` column here, which should point to none, one or many fields of the `organization` model.
This column specifies which additional information the organization has to specify on their page if they choose this type.